src/deployment/deployments.controller.ts

```diff
@@ -1,9 +1,25 @@
-import {Controller, Post} from '@nestjs/common';
+import {Controller, Get, HttpCode, HttpStatus, Post, Req} from '@nestjs/common';
+import {DynamoDBDeploymentsRepository} from '../repository/dynamodb/DynamoDBDeploymentsRepository';
 
 @Controller('/v1/deployments')
 export class DeploymentsController {
+  constructor(
+    private readonly repository: DynamoDBDeploymentsRepository
```

Comment:
Would be great to inject the interface ...

Reply:
Yes